composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "php": "^5.3 || ^7.0",
+        "php": "^7.0",
```

Comment:
As he is very busy right now. Maybe we can merge this and bump the version to 7.1 afterwards?